src/axom/spin/UniformGrid.hpp

```diff
@@ -515,9 +514,15 @@ void UniformGrid<T, NDIMS, ExecSpace, StoragePolicy>::initialize(axom::ArrayView
   axom::for_all<ExecSpace>(
     bboxes.size(),
     AXOM_LAMBDA(IndexType idx) {
-      const BoxType bbox = bboxes[idx];
+      const BoxType& bbox = bboxes[idx];
+      if(!bbox.isValid())
+      {
+        return;
+      }
```

Comment:
This check ensures that we do not try to insert geometry associated with invalid bounding boxes into our `UniformGrid`